scripts/test.sh

```diff
@@ -333,49 +333,70 @@ case "$TARGET" in
   RUN_STITCH_IN_FORGROUND=true
   start_server
   ;;
-"react-tests-android")
+"test-android")
+  npm ci --ignore-scripts
```

Comment:
This might depend on our common understanding of the purpose of the `test.sh` script:
- If it's only intended for use on CI, this chance is okay, since we control how its called from the Jenkinsfile.
- If it's supposed to be used locally, I don't think we should use `npm install` and especially not `npm ci` here, since it makes each iteration on tests / bugs take much longer to validate. Is there a less demanding script that we can call to get the intended behaviour?

Reply:
This is used to run the android scripts locally and on ci. This PR targets improving on the situation where we didn't have android tests running at all. I would suggest creating an issue and doing a following PR for changes which allow for better develop iterations. Which might take into account our future plans on moving away from this script to something better.

Reply:
Can't we just move this line out into the Jenkinsfile? Then I would be happy 👍

Reply:
that would break the local runs of this script. Unfortunately we have this script which tries to provide both local and CI runs. Maybe our next solution to this would be more robust. 
What I have chosen is to do is do all scripts in JavaScript. This is universal for us since we have Node everywhere. So it should run on every OS for example unlike the shell differences between Linux and Mac that we experience currently. And is more easily debugged and fixed by everyone when needed. Hence my `build-android.js` here. Also my opinion is that the Jenkins script should only be invoking these JS scripts the same way we are running them locally. This way we will not find ourselves in the situation to have big discrepancies between how Jenkins and local scripts are run.